cfgov/unprocessed/apps/regulations3k/css/reg-navigation.less

```diff
@@ -3,65 +3,118 @@
    Custom eRegs secondary navigation styles
    ========================================================================== */
 
-.o-secondary-navigation {
-    &__no-children {
-        .respond-to-max( @bp-sm-max, {
-            display: block;
-        } );
-    }
+// Regs3K sidebar
+.o-regs3k-sidebar {
+    .respond-to-max( @bp-sm-max {
+        background-color: @gray-5;
+        border-bottom: 1px solid @gray-40;
+        border-top: 1px solid @gray-40;
+        margin-bottom: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em );
+    } );
+}
 
-    .o-expandable {
-        margin-bottom: 0;
-    }
+// Regs3K secondary nav sections
+.o-regs3k-sections {
 
-    // TODO: If secondary navigation is overriding o-expandables appearance,
```

Comment:
This is still an applicable TODO, isn't it?

Reply:
ehhhh, it's not really using expandables for the main outer expandable, but yeah, it should probably be figured out a bit later.